addOns/selenium/src/main/java/org/zaproxy/zap/extension/selenium/ExtensionSelenium.java

```diff
@@ -187,6 +193,26 @@ public void hook(ExtensionHook extensionHook) {
         }
 
         extensionHook.addApiImplementor(seleniumApi);
+
+        ExtensionScript extScript =
```

Comment:
Could be moved to an instance variable to avoid getting the extension each time in `getWebDriverImpl`.

Reply:
Done